src/docs/property_view.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Property View
+---
+
+# Property View
+
+Many IDEs, such as the traditional Eclipse IDE, have the notion of a global, extensible property view,
+which shows additional information about the current selection within the IDE.
+Property views are heavily used in those IDEs for showing details of elements in e.g. diagram editors,
+complex tree editors, or the file explorer selection.
+Therefore, the main idea is to have a global, generic property view in the IDE,
+but allow specific implementations to extend the contents of the global property view
+with specific additional information for some type of selection.
+
+The `@theia/property-view extension` contributes a generic, global property view based on Theia's global selection.
+The property view widget can be opened/toggled either via menu `View->Properties` or via shortcut <kbd>Shift</kbd>+<kbd>Alt</kbd>+<kbd>P</kbd>.
+It is located in the bottom dock area per default.
+The following two default content widgets are implemented in this extension:
+
+`EmptyPropertyViewWidget`: If no other widget can be provided, a simple message (`No properties available`) is shown.
+`ResourcePropertyViewWidget`: Displays additional information about a file (e.g. location, name, last modified) that is selected in the file explorer or in the active monaco editor.
```

Comment:
Just to illustrate better that these are the two widgets, we can have them in lists:

```suggestion
- `EmptyPropertyViewWidget`: If no other widget can be provided, a simple message (`No properties available`) is shown.
- `ResourcePropertyViewWidget`: Displays additional information about a file (e.g. location, name, last modified) that is selected in the file explorer or in the active monaco editor.
```